lib/blkproto.ml

```diff
@@ -37,7 +37,7 @@ module Mode = struct
   let to_string = function
     | ReadOnly -> "r"
     | ReadWrite -> "w"
-  let of_string = function
+  let _of_string = function
```

Comment:
Are these functions unused? I guess we should either delete them or expose them in the API.

Reply:
I'd lean toward deleting them, personally.

Reply:
Deleting them seems good. Can be a separate PR though since this one is only about exposing the minimal mli